### PR TITLE
Versions update, Layer manager extension

### PR DESCRIPTION
**Pull request by SLeitgeb**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -100,6 +100,7 @@
     "karma-jasmine-html-reporter": "^1.7.0",
     "prettier": "^2.3.2",
     "ng-packagr": "^11.2.4",
+    "prettier": "^2.3.1",
```
Comment (raitisbe) [inline package.json:103]: Prettier seems to be listed twice

File: projects/hslayers-material/package.json [hunk 2]
```diff
@@ -3,18 +3,18 @@
...
+    "@angular/cdk": "^11.2.13",
+    "@angular/common": "^11.2.13",
+    "@angular/core": "^11.2.13",
+    "@angular/forms": "^11.2.13",
+    "@angular/localize": "^11.2.13",
+    "@angular/material": "^11.2.13",
+    "ol": "^6.5.0",
     "ol-popup": "^4.0.0",
-    "hslayers-ng": "^3.0.0"
+    "hslayers-ng": "^4.0.1"
```
Comment (raitisbe) [inline projects/hslayers-material/package.json:15]: Can target 5 now

**Review by raitisbe — CHANGES_REQUESTED** [review]

**Comment by SLeitgeb** [comment]
Added an option to specify layout widths with percentage values in d05b2ecbd5861b96145bc789dff7edb6c4bc7d77

**Review by raitisbe — APPROVED** [review]

**Comment by raitisbe** [comment]
Thank you. Just a quick thing before merge, could you rename the commits and squash some of them if possible to follow https://www.conventionalcommits.org/en/v1.0.0/ standard?

**Comment by SLeitgeb** [comment]
This should be much better hopefully! Thanks for the link, I was wondering where those neat commit messages came from :)

**Comment by raitisbe** [comment]
> This should be much better hopefully! Thanks for the link, I was wondering where those neat commit messages came from :)

You can also create the commit with `npm run commit` using a wizard ;)

**Pull request merged**